src/components/asset-list/RecyclerAssetList2/WrappedNFT.tsx

```diff
@@ -6,9 +6,11 @@ import { useNavigation } from '@rainbow-me/navigation';
 import Routes from '@rainbow-me/routes';
 
 export default React.memo(function WrappedNFT({
+  onPress,
   uniqueId,
   placement,
 }: {
+  onPress: (asset: any) => void;
```

Comment:
this could be undefined as well? just saying it for the type and the `onPress || handleItemPress` check below